activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -626,9 +627,19 @@ def collation
       end
 
       def tables(name = nil) # :nodoc:
+        ActiveSupport::Deprecation.warn(<<-MSG.squish)
+          #tables of mysql adapter returns tables and views,
```

Comment:
What about the following wording:

>         #tables currently returns both tables and views.
        This behavior is deprecated and will be changed with Rails 5.1 to only return tables.
        Use #data_sources instead.